cli/src/api/endpoints.rs

```diff
@@ -191,6 +200,10 @@ fn get_api_path(api_uri: &str) -> Result<Url, BaseUriError> {
     Ok(parse_base_url(api_uri)?.join(API_PATH)?)
 }
 
+fn get_locksmith_path(api_uri: &str) -> Result<Url, BaseUriError> {
```

Comment:
Might as well use this for `locksmith_discovery()` above as well

Reply:
You mean to avoid the proxy? I suppose that should work just fine.